tests/foreman/cli/test_syncplan.py

```diff
@@ -99,15 +99,17 @@ def valid_name_interval_update_tests():
     ]
 
 
-def validate_task_status(repo_id, max_tries=6):
+def validate_task_status(repo_id, module_org_id, max_tries=6):
```

Comment:
no need to keep `module_org_id`explicitly, we can just say `org_id`. Anyways the reference is passed is module_org.id

Reply:
Ok, fix incoming, please wait for test results